src/components/Trade/TradeInput.tsx

```diff
@@ -198,6 +199,14 @@ export const TradeInput = () => {
     [updateAction, updateIsSendMax, updateAmount, handleInputAmountChange],
   )
 
+  /*
+     On initial load we are missing a swapper, among other things, which causes invalid errors and warning to be shown
+     to the user. We want to wait until we've started making network calls before validating the state.
+   */
```

Comment:
I also have concerns about these heuristics, regardless of whether the initial load state lives in a singleton or in a reactive instance (e.g a hook/a component or whatever might have different instances):

It looks like the intent is really to detect that we've started making network calls, e.g there's any network request, either pending, fulfilled, or rejected. `uninitialized` should work too, since it's only used in case of a `{skip: true}` RTK query hooks option which we don't seem to use anywhere across the `swapperApi` domain. In other words, if `swapperApi.queries` has any member, then we've started making network calls.

In this case, would it make sense to have a new `swapperApi.queries` selector to achieve the same effect as this e.g `Object.keys(state.swapperApi.queries).length`? It would be 0-length on the *first* load of the *first* swapper instance, and have a truthy length as soon as said instance first mounts and start making requests.